resources/lib/services/nfsession/nfsession.py

```diff
@@ -296,7 +296,6 @@ def perpetual_path_request(self, paths, length_params, perpetual_range_start=Non
                              .format(response_type))
             else:
                 range_end = range_start + request_size
-                break
```

Comment:
Good catch, there's indeed a runaway break there :-(
https://github.com/CastagnaIT/plugin.video.netflix/commit/bf109727e1f253b0fea1484f3f6280fb11e1be26

Reply:
I found another issue introduced by that PR. (Everything else looks fine after reviewing)

Reply:
> I found another issue introduced by that PR. (Everything else looks fine after reviewing)

Either a word is missing, either I there's really an another issue and I'm curious